hooks/post-command

```diff
@@ -31,6 +31,17 @@ main() {
     exit 0
   fi
 
+  curl -s https://codecov.io/bash > codecov;
```

Comment:
it would be nice to download the script only once here, verify the checksum and then use it inside a docker container. Otherwise we download it twice, which doesn't guarantee that the second time we downloaded the same script as the first time.

Reply:
True, I just need to figure out where it downloads it to so it can be mounted into the docker container